### PR TITLE
settings: allow admin to change credentials.

### PR DESCRIPTION
**Pull request by krishnasrinivas**

### CODE REVIEW

File: app/js/components/Browse.js [hunk 1]
```diff
@@ -480,7 +531,7 @@ export default class Browse extends React.Component {
         let used = total - free
         let usedPercent = (used / total) * 100+'%'
         let freePercent = free * 100 / total
-
+        console.log("rendering", settings)
         return (
```
Comment (harshavardhana) [inline app/js/components/Browse.js]: is this needed?

File: app/js/web.js [hunk 2]
```diff
@@ -96,4 +96,14 @@ export default class Web {
...
+    return this.makeCall('GenerateAuth')
+  }
+  SetAuth(args) {
+    return this.makeCall('SetAuth', args)
+                .then(res => {
+                  localStorage.token = `${res.token}`
+                  return res
+                })
+  }
 }
```
Comment (harshavardhana) [inline app/js/web.js]: Do not build ui-assets.go - build it with another commit.

Comment (krishnasrinivas) [inline app/js/web.js]: ui-assets.go should be added to .gitignore. we can use -f to add it when we make a release.

**Comment by harshavardhana** [comment]
> ui-assets.go should be added to .gitignore. we can use -f to add it when we make a release.

Yeah go ahead and do that. Also add an entry in README.md

**Comment by harshavardhana** [comment]
LGTM

**Pull request merged**